src/main/scala/org/graphframes/lib/ParallelPersonalizedPageRank.scala

```diff
@@ -0,0 +1,89 @@
+/*
+ * Licensed to the Apache Software Foundation (ASF) under one or more
+ * contributor license agreements.  See the NOTICE file distributed with
+ * this work for additional information regarding copyright ownership.
+ * The ASF licenses this file to You under the Apache License, Version 2.0
+ * (the "License"); you may not use this file except in compliance with
+ * the License.  You may obtain a copy of the License at
+ *
+ *    http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+package org.graphframes.lib
+
+import org.apache.spark.graphx.{lib => graphxlib}
+import org.graphframes.{GraphFrame, Logging}
+
+class ParallelPersonalizedPageRank private[graphframes] (
```

Comment:
Let me get another PR for the documentation. I think it requires some modification and merging from the `GraphX` main doc and the `ParallelPersonalizedPageRank` function doc